rollbar_dart/lib/src/sender/http_sender.dart

```diff
@@ -39,6 +39,13 @@ class HttpSender implements Sender {
   @override
   Future<bool> sendString(String payload) async {
     try {
+      if (_State.suspended) {
+        throw HttpException(
```

Comment:
we don't really need to throw here. the suspension is a legit expected conditional state and not really an exceptional event. probably, better to just return `false` instead.

Reply:
Keep in mind this is gonna disappear when I do the throttling stuff we talked about.